client/src/App.tsx

```diff
@@ -0,0 +1,13 @@
+import GlobalStyles from './box/global'
+import Header from './components/header'
+
+const App = (): JSX.Element => {

```

Comment:
you can write like this if you don't pass props: 

```suggestion
const App = () => {
```

Reply:
Thanks. I will write such as you propused.